docs/source/api/v3/deliveryservices_required_capabilities.rst

```diff
@@ -19,6 +19,8 @@
 ``deliveryservices_required_capabilities``
 ******************************************
 
+.. deprecated:: ATCv7
```

Comment:
A few notes on these deprecation notices

- all of APIv3 is deprecated, so it's superfluous to note that for any specific endpoint/method thereof
- If an entire endpoint is being deprecated, you don't also need to individually deprecate each request method it supports
- Generally you want to specify the _API_ version that something was deprecated, unless it's not possible to do that for some reason or it just makes more sense to list the ATC version instead.
- Ideally the notice will come with some information about why it's being deprecated and/or what to use instead. This makes generating changelogs using Sphinx much more useful.
-